protocol-designer/src/organisms/EditNickNameModal/index.tsx

```diff
@@ -53,19 +53,27 @@ export function EditNickNameModal(props: EditNickNameModalProps): JSX.Element {
           >
             {t('shared:cancel')}
           </SecondaryButton>
-          <PrimaryButton onClick={saveNickname}>
+          <PrimaryButton
+            onClick={saveNickname}
+            disabled={nickName.length >= 115}
```

Comment:
let's define a const for the max value

```suggestion
const MAX_TEXT_INPUT = 115

            disabled={nickName.length >= MAX_TEXT_INPUT}
```